src/java.desktop/macosx/classes/sun/lwawt/macosx/LWCToolkit.java

```diff
@@ -146,24 +144,14 @@ public final class LWCToolkit extends LWToolkit {
     static {
         System.err.flush();
 
-        @SuppressWarnings({"removal", "restricted"})
-        ResourceBundle platformResources = java.security.AccessController.doPrivileged(
-                new java.security.PrivilegedAction<ResourceBundle>() {
-            @Override
-            public ResourceBundle run() {
-                ResourceBundle platformResources = null;
-                try {
-                    platformResources = ResourceBundle.getBundle("sun.awt.resources.awtosx");
-                } catch (MissingResourceException e) {
-                    // No resource file; defaults will be used.
-                }
-
-                System.loadLibrary("awt");
-                System.loadLibrary("fontmanager");
-
-                return platformResources;
-            }
-        });
+        ResourceBundle platformResources = null;
+        try {
+            platformResources = ResourceBundle.getBundle("sun.awt.resources.awtosx");
+        } catch (MissingResourceException e) {
+            // No resource file; defaults will be used.
+        }
+
+        loadLibrary();
```

Comment:
I think what you are doing is fine.

Unrelated, that System.err.flush() seems odd. I've filed a bug to look at it afterwards since it has NOTHING to do with the doPrivileged.